sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -211,7 +267,7 @@ def _get_available_requests_columns(self, cursor, all_expected_columns):
 
     @staticmethod
     def _get_sort_key(r):
-        return r.get("query_start") or datetime.datetime.now()
+        return r.get("query_start") or datetime.datetime.now().isoformat()
```

Comment:
That's right. The `query_start` is `None` for idle blockers because there's no associated requests. The query cast timestamp to style `126`, which is the i[so format](https://learn.microsoft.com/en-us/sql/t-sql/functions/cast-and-convert-transact-sql?view=sql-server-ver16) `126 4 | ISO8601 | yyyy-mm-ddThh:mi:ss.mmm (no spaces)` according to MS doc.